lint_test.go

```diff
@@ -24,7 +24,7 @@ func TestLint(t *testing.T) {
 		// Ignore all errors from unused.go
 		lint.RegexpMatch(`unused\.go`),
 		// Ignore duplicates we're okay with.
-		dupl.SkipTwo, dupl.Skip("golint.go:1,12"))
+		dupl.SkipTwo, dupl.Skip("golint.go:1,12"), dupl.Skip("errcheck.go:17,19"))
```

Comment:
Fails like this if not ignored:
```
$ go test
--- FAIL: TestLint (3.48s)
        lint_test.go:30: dupl.Check: found 4 clones:
                  /home/aatos/go/src/github.com/surullabs/lint/errcheck/errcheck.go:17,19
                  /home/aatos/go/src/github.com/surullabs/lint/gosimple/gosimple.go:15,17
                  /home/aatos/go/src/github.com/surullabs/lint/gostaticcheck/gostaticcheck.go:15,17
                  /home/aatos/go/src/github.com/surullabs/lint/structcheck/structcheck.go:17,21
                
FAIL
exit status 1
FAIL    github.com/surullabs/lint       7.253s
```

so the `Check` function in those checkers: https://github.com/surullabs/lint/pull/10/files#diff-b311a1952f43252ba461208d650af691R15